operator/cmd/actions/start.go

```diff
@@ -68,15 +78,35 @@ func operatorMain(ctx *cli.Context) error {
 	endpoint := operatorConfig.Operator.OperatorTrackerIpPortAddress + "/versions"
 	operator.Logger.Info("Sending version to operator tracker server: ", "endpoint", endpoint)
 
-	res, err := http.Post(endpoint, "application/json",
-		bodyBuffer)
+	res, err := http.Post(endpoint, "application/json", bodyBuffer)
 	if err != nil {
 		// Dont prevent operator from starting if operator tracker server is down
 		operator.Logger.Warn("Error sending version to metrics server: ", "err", err)
 	} else if res.StatusCode != http.StatusCreated && res.StatusCode != http.StatusNoContent {
 		operator.Logger.Warn("Error sending version to operator tracker server: ", "status_code", res.StatusCode)
 	}
 
+	return nil
+}
+
+func operatorMain(ctx *cli.Context) error {
+	operatorConfigFilePath := ctx.String("config")
+	operatorConfig := config.NewOperatorConfig(operatorConfigFilePath)
+	err := sdkutils.ReadYamlConfig(operatorConfigFilePath, &operatorConfig)
+	if err != nil {
+		return err
+	}
+
+	operator, err := operator.NewOperatorFromConfig(*operatorConfig)
+	if err != nil {
+		return err
+	}
+
+	err = updateTelemetryService(operator, ctx, operatorConfig)
```

Comment:
```suggestion
	err = operator.sendTelemetryData(ctx, operatorConfig)
```

Reply:
I think even the `operatorConfig` argument can be omitted if we define `sendTelemetryData` as a method of Operator